### ISSUE TITLE
Storage blobs break blob

### ISSUE THREAD

**Issue by Kyep** [ParadiseSS13/Paradise#9365]
**Problem Description**:
Blobs that buy a storage blob can end up with -50, or 0, resources, and be unable to do anything.

**What did you expect to happen**: 
Storage blobs to simply increase storage capacity, without affecting resource count.

**What happened instead**:
Placing one ruined that blob's day.

**Why is this bad/What are the consequences:**
Storage blobs are bugged to the point that placing them can get you killed.

**Steps to reproduce the problem**:
Be a blob. Place a storage blob or two.

**Possibly related stuff**
Storage blobs serve no purpose. There's no advantage to hoarding resources past 100 anyway. Storage blobs have no good reason to be in the game. Simply removing them would be fine.

**Comment by Birdtalon** [comment]
I've played storage blobs before with no issues. 🤔 I'll try to reproduce.

**Comment by Taaketa** [comment]
Storage Blobs have a purpose. They cut down on the quantity of the station that needs to be covered by Blob before the round ends.

**Comment by Kyep** [comment]
Did not know that. Assuming its true... yeah, worth keeping.

**Comment by Taaketa** [comment]
Once won a blob round by having 10 resource nodes and plonking done storage nodes where I could. I only had lower Sci maintenance from solar to turbine covered in Blob and won. Much to everyone's dismay as they thought they were beating me back. (When I was dumping my res into storage)

**Comment by Birdtalon** [comment]
>Storage Blobs have a purpose. They cut down on the quantity of the station that needs to be covered by Blob before the round ends.

That's complete nonsense do you have some evidence for this?

**Comment by Birdtalon** [comment]
Spawned as a blob. Placed multiple storage blobs. Unable to reproduce. Resources generated as usual.

**Comment by Taaketa** [comment]
> That's complete nonsense do you have some evidence for this?

@Birdtalon  Seems I got the scaling confused with the amount of area needed to be claimed based on server pop. at the time Blob rounds start. But next time, perhaps don't challenge someone like a dick.

**Comment by Birdtalon** [comment]
@Taaketa asking you for evidence to back up your claim is being a dick? Okay sure.

**Comment by Crazylemon64** [comment]
@Birdtalon You could have done better to phrase your request without the "complete nonsense" qualifier - language like that makes exchanges heated.

**Comment by Rurik123** [comment]
The bug happened to me in the past, not when I placed the storage blob, but when it was destroyed. After it was destroyed, all resource count ceased, and I had no further income; thus keeping my resource points at zero. As a note, I managed to catch my lack of income early once, with a sizable amount of points left. As a result, If I recall correctly, rebuying the storage node after its destruction reversed the affects and allowed income once more. 

I hope that information is helpful in some way.
